sleap/io/cameras.py

```diff
@@ -449,6 +450,7 @@ class InstanceGroup:
     _dummy_instance: Optional[Instance] = field(default=None)
     camera_cluster: Optional[CameraCluster] = field(default=None)
     _score: Optional[float] = field(default=None)
+    triangulated_3d_pose: Optional[np.ndarray] = field(default=None)
```

Comment:
```suggestion
    _triangulation: Optional[np.ndarray] = field(default=None)
```

Let's make this a psuedo-private method by prefixing with "_". This prefixed underscore tells developers that this method either is only intended to be used (or only used) internally within the containing class. This also allows us to name this underlying attribute as `_<name-of-referencing-property-method>`.